### ISSUE TITLE
About training time and epoch

### ISSUE THREAD

**Issue by weiren1998** [pmm09c/ntire-dehazing#4]
I want to know if your training epoch is setting 3000? Is it reasonable?
And how long will you spend on training a FastNet model on NYU dataset?
Thank you very much!!

**Comment by pmm09c** [comment]
I believe we never ran for more than ~300 or so epochs. We just set the training time to 3000 so it would keep running when we left it training overnight. The code should "overfit" to the validation loss and keep writing out weights every-time the loss is lowered.
